packages/falso/src/lib/collection.ts

```diff
@@ -14,7 +14,7 @@ import { FakeOptions, fake } from './core/core';
  *
  */
 
-export function toCollection<Collection, Options extends FakeOptions = never>(
+export function toCollection<Collection, Options extends FakeOptions = {length: number}>(
```

Comment:
please add a test in collection.spec

Reply:
@NetanelBasal hopefully that test is sufficient?